libs/design/src/atoms/radio/radio.component.ts

```diff
@@ -0,0 +1,55 @@
+import { Component, OnInit, Input, ElementRef, Renderer2, HostBinding, ViewEncapsulation, ChangeDetectionStrategy, HostListener } from '@angular/core';
+
+
+@Component({
+  // tslint:disable-next-line: component-selector
+  selector: 'input[daff-radio]',
+  templateUrl: './radio.html',
+  styleUrls: ['./radio.scss'],
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class DaffRadioComponent implements OnInit {
```

Comment:
@Nolan-Arnold can you propose an interface on how you'd like to use this with a FormControl?